CMakeLists.txt

```diff
@@ -45,12 +45,8 @@ include( maptk-depends )
 #
 include(CMakeDependentOption)
 
-# we can't build shared libraries on Windows so we leave it off by default
-if (WIN32)
-  option(BUILD_SHARED_LIBS "Build shared libraries." ON)
-else()
-  option(BUILD_SHARED_LIBS "Build shared libraries." OFF)
-endif()
+# Always build shared libraries, required due to plugin infrastructure
+set(BUILD_SHARED_LIBS ON)
```

Comment:
I think this still lets a user turn it off?  Might want to force setting into cache instead of / in addition to this.